src/tora/train/risk/GUI/StaticInformations.java

```diff
@@ -10,6 +10,12 @@
 /**
  * Created by intern on 7/20/15.
  */
+
+/**
+ * StaticInformations e un fel de singleton, doar mai rau. Garantat sa nu stii cine scrie si cine citeste. Cand face
+ * asta, sau de ce. Clasa asta trebuie sa dispara. Cautati o modalitate in care sa comunicati informatiile acestea
```

Comment:
Clasa asta am facut-o doar pentru verificarea logicii prin swing (in cazul in care nu reusim sa terminam partea client-server), ea nu va exista la final; informatiile respective vor fi retinute in server daca bine am inteles.